ssl/quic/quic_impl.c

```diff
@@ -4628,6 +4628,71 @@ int ossl_quic_listen(SSL *ssl)
     return ret;
 }
 
+QUIC_TAKES_LOCK
+int ossl_quic_peeloff_conn(SSL *listener, SSL *new_conn)
+{
+    QCTX lctx;
+    QCTX cctx;
+    QUIC_CHANNEL *new_ch;
+    QUIC_CONNECTION *qc = NULL;
+    QUIC_LISTENER *ql = NULL;
+    SSL *tls = NULL;
+    int ret = 0;
+
+    if (!expect_quic_listener(listener, &lctx))
+        return 0;
```

Comment:
are return values 0, -1, 1 needed here? the description of return value is missing in SSL_new_listener.pod manual page. so I don't know, therefore I'm asking.